lib/generators/cloopen/templates/cloopen_setup.rb

```diff
@@ -2,13 +2,13 @@
 
 Cloopen.account_sid = "YOUR_ACCOUNT_SID"
 Cloopen.auth_token  = "YOUR_AUTH_TOKEN"
-Cloopen.app_id = "YOUR_APP_ID"
-Cloopen.env = "production"
+Cloopen.app_id      = "YOUR_APP_ID"
+Cloopen.env         = "production"
```

Comment:
No.


I already merge your pull request, and released the new version.


Great work, Thanks.


-- 
Ryan, Lv


On June 6, 2015 at 11:56:19, debbbbie (notifications@github.com) wrote:

In lib/generators/cloopen/templates/cloopen_setup.rb:

> @@ -2,13 +2,13 @@
>   
>  Cloopen.account_sid = "YOUR_ACCOUNT_SID"
>  Cloopen.auth_token  = "YOUR_AUTH_TOKEN"
> -Cloopen.app_id = "YOUR_APP_ID"
> -Cloopen.env = "production"
> +Cloopen.app_id      = "YOUR_APP_ID"
> +Cloopen.env         = "production"
Need I remove it?

—
Reply to this email directly or view it on GitHub.